core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -218,19 +218,56 @@
       "${alluxio.worker.memory.size}"),
+  /**
+   * @deprecated It will be removed in 2.0.0.
+   * Use {@link #WORKER_TIERED_STORE_LEVEL0_HIGH_WATERMARK_RATIO} and
+   * {@link #WORKER_TIERED_STORE_LEVEL0_LOW_WATERMARK_RATIO} instead.
+   */
+  @Deprecated
   WORKER_TIERED_STORE_LEVEL0_RESERVED_RATIO(Name.WORKER_TIERED_STORE_LEVEL0_RESERVED_RATIO, "0.1"),
+  WORKER_TIERED_STORE_LEVEL0_HIGH_WATERMARK_RATIO(
+      Name.WORKER_TIERED_STORE_LEVEL0_HIGH_WATERMARK_RATIO, 1.0),
+  WORKER_TIERED_STORE_LEVEL0_LOW_WATERMARK_RATIO(
+      Name.WORKER_TIERED_STORE_LEVEL0_LOW_WATERMARK_RATIO, 0.7),
   WORKER_TIERED_STORE_LEVEL1_ALIAS(Name.WORKER_TIERED_STORE_LEVEL1_ALIAS, null),
   WORKER_TIERED_STORE_LEVEL1_DIRS_PATH(Name.WORKER_TIERED_STORE_LEVEL1_DIRS_PATH, null),
   WORKER_TIERED_STORE_LEVEL1_DIRS_QUOTA(Name.WORKER_TIERED_STORE_LEVEL1_DIRS_QUOTA, null),
-  WORKER_TIERED_STORE_LEVEL1_RESERVED_RATIO(Name.WORKER_TIERED_STORE_LEVEL1_RESERVED_RATIO, null),
+  /**
+   * @deprecated It will be removed in 2.0.0.
+   * Use {@link #WORKER_TIERED_STORE_LEVEL1_HIGH_WATERMARK_RATIO} and
+   * {@link #WORKER_TIERED_STORE_LEVEL1_LOW_WATERMARK_RATIO} instead.
+   */
+  @Deprecated
+  WORKER_TIERED_STORE_LEVEL1_RESERVED_RATIO(Name.WORKER_TIERED_STORE_LEVEL1_RESERVED_RATIO, "0.1"),
+  WORKER_TIERED_STORE_LEVEL1_HIGH_WATERMARK_RATIO(
+      Name.WORKER_TIERED_STORE_LEVEL1_HIGH_WATERMARK_RATIO, 1.0),
+  WORKER_TIERED_STORE_LEVEL1_LOW_WATERMARK_RATIO(
+      Name.WORKER_TIERED_STORE_LEVEL1_LOW_WATERMARK_RATIO, 0.7),
   WORKER_TIERED_STORE_LEVEL2_ALIAS(Name.WORKER_TIERED_STORE_LEVEL2_ALIAS, null),
   WORKER_TIERED_STORE_LEVEL2_DIRS_PATH(Name.WORKER_TIERED_STORE_LEVEL2_DIRS_PATH, null),
   WORKER_TIERED_STORE_LEVEL2_DIRS_QUOTA(Name.WORKER_TIERED_STORE_LEVEL2_DIRS_QUOTA, null),
-  WORKER_TIERED_STORE_LEVEL2_RESERVED_RATIO(Name.WORKER_TIERED_STORE_LEVEL2_RESERVED_RATIO, null),
+  /**
+   * @deprecated It will be removed in 2.0.0.
+   * Use {@link #WORKER_TIERED_STORE_LEVEL2_HIGH_WATERMARK_RATIO} and
+   * {@link #WORKER_TIERED_STORE_LEVEL2_LOW_WATERMARK_RATIO} instead.
+   */
+  @Deprecated
+  WORKER_TIERED_STORE_LEVEL2_RESERVED_RATIO(Name.WORKER_TIERED_STORE_LEVEL2_RESERVED_RATIO, "0.1"),
+  WORKER_TIERED_STORE_LEVEL2_HIGH_WATERMARK_RATIO(
+      Name.WORKER_TIERED_STORE_LEVEL2_HIGH_WATERMARK_RATIO, 1.0),
+  WORKER_TIERED_STORE_LEVEL2_LOW_WATERMARK_RATIO(
+      Name.WORKER_TIERED_STORE_LEVEL2_LOW_WATERMARK_RATIO, 0.7),
   WORKER_TIERED_STORE_LEVELS(Name.WORKER_TIERED_STORE_LEVELS, 1),
   WORKER_TIERED_STORE_RESERVER_ENABLED(Name.WORKER_TIERED_STORE_RESERVER_ENABLED, false),
   WORKER_TIERED_STORE_RESERVER_INTERVAL_MS(Name.WORKER_TIERED_STORE_RESERVER_INTERVAL_MS, 1000),
+  /**
+   * @deprecated It will be removed in 2.0.0.
+   * If you want use new version space reserver (watermark), you must turn on this switch.
+   */
+  @Deprecated
+  WORKER_TIERED_STORE_RESERVER_NEWVERSION(Name.WORKER_TIERED_STORE_RESERVER_NEWVERSION, false),
```

Comment:
You don't need to explicitly have a field here. It's sufficient to check if the user has specified the old `RESERVED_RATIO` fields. If they have and have not specified the new `HIGH/LOW` watermarks, log a warning message informing them that this is deprecated and switch to the new behavior. For this reason, you may want to leave the `HIGH/LOW` watermarks without a default since space reserver is not enabled by default.